misk/web/tabs/admin-dashboard/package.json

```diff
@@ -15,7 +15,7 @@
     "prebuild": "npm run-script lint",
     "reinstall": "rm -rf node_modules && npm run-script install",
     "start": "npm run-script prebuild && cross-env NODE_ENV=development webpack-dev-server",
-    "test": "jest --passWithNoTests --maxWorkers=2",
+    "test": "jest --passWithNoTests",
```

Comment:
Can you re-run the `miskweb ci-build -e` step and make sure your local Misk Web CLI is up to date? I suspect this diff suggests it is not.

Update: `miskweb update` or `npm i -g @misk/cli`